### PR TITLE
SVG Graph and CrunchLab

### PR DESCRIPTION
**Pull request by KernelDeimos**
PR adds a Graph model, and an SVG graph renderer. A u2 controller called CrunchLab is added to this pr as an example of rendering the graph.

### CODE REVIEW

File: src/foam/graph/graph.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/graph/graph.js]: license header please.

**Review by jlhughes — COMMENTED** [review]
Can you move all these classes into their own files. Otherwise it's impossible to debug in the browser.

File: src/foam/graph/graph.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/**
```
Comment (jlhughes) [inline src/foam/graph/graph.js]: Thanks for splitting. One last request; to be consistent, can you Capitalize this source file please.

File: src/foam/u2/crunch/lab/CrunchLab.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/u2/crunch/lab/CrunchLab.js:7]: license headers please.

File: src/foam/graph/graph.js (hunk 2 above)
Comment (KernelDeimos) [reply src/foam/graph/graph.js]: Looks like Git didn't let me do it the first time. Found a way around it

**Review by jlhughes — APPROVED** [review]

**Pull request merged**